.github/mongo_setup.sh

```diff
@@ -0,0 +1,15 @@
+docker run -d -p 27001:27001 -p 27002:27002 -p 27003:27003 --name mongo -e "REPLICA_SET_NAME=rs0" zxilly/mongodb-rs
```

Comment:
Since you do not have the docker container documented, nor use any common one, maybe add the docker file into this repo? Now you kinda have to run the container locally and be at the mercy of the creator. Also easier to manage everything from one place.

What do you think @hsluoyz

Reply:
I have change this. now it will build image from Dockerfile

Reply:
Dockerfile is good. Thank both!